<launch>

   <rosparam file="$(find uta_pr2_forceControl)/pr2_controller.yaml"
             command="load" />

   <node pkg="pr2_controller_manager"
         type="spawner"
         args="pr2_neuroadptController"
         name="pr2_neuroadptController_spawner" />

</launch>
